pkg/server/ca/ca_test.go

```diff
@@ -241,8 +238,8 @@ func (s *CATestSuite) TestSignX509SVIDCapsTTLToCATTL() {
 }
 
 func (s *CATestSuite) TestSignX509SVIDValidatesTrustDomain() {
-	_, err := s.ca.SignX509SVID(ctx, s.createX509SVIDParamsInDomain("foo.com"))
-	s.Require().EqualError(err, `"spiffe://foo.com/workload" does not belong to trust domain "example.org"`)
+	_, err := s.ca.SignX509SVID(ctx, s.createX509SVIDParamsInDomain(spiffeid.RequireTrustDomainFromString("foo.com")))
```

Comment:
All of these repeated `Require*` method calls could be replaced with a package level variable.